Add `osu!taiko World Cup 2024: Semifinals Recap` news post (#11264)
ppy/osu-wiki: news/2024/2024-04-17-osutaiko-world-cup-2024-semifinals-recap.md

---
layout: post
title: "osu!taiko World Cup 2024: Semifinals Recap"
date: 2024-04-17 22:00:00 +0000
---

Catch up on the latest and greatest from the previous week of the osu!taiko World Cup!

![](/wiki/Tournaments/TWC/2024/img/twc2024-banner.jpg)

> The brackets are slowly solidifying, and with that we are getting a glimpse of what the final few weeks of the tournament will look like. While there is only 1 match we'd like to showcase this week, it's a banger worth your time, whether you just read the recap, or watch the VOD of the entire match.

Reviewer: ```suggestion
The brackets are slowly solidifying, and with that we are getting a glimpse of what the final few weeks of the tournament will look like. While there is only 1 match we'd like to showcase this week, it's a banger worth your time, whether you're watching the entire match VOD or settling for a recap.
```

You know the drill: first we cover the mappool and the new [osu! originals](/wiki/osu!_originals), then the full results, then we jump onto the highlights.

## Mappool showcase

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

[**Download the pool here! (118 MB)**](https://osu.ppy.sh/beatmaps/packs/P243)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=v2113703930&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> The heat is slowly being turned up, and the mappool is starting to transform into the type of high-level monstrosity you've come to expect from a World Cup. The structure hasn't changed as much as its hit the point in the difficulty charts where higher seeded teams that ended up in the losers' brackets are starting to liquidate their opponents in order to try and secure a spot in the Finals.

Reviewer: ```suggestion
The heat is slowly being turned up, and the mappool is starting to transform into the type of high-level monstrosity you've come to expect from a World Cup. The structure hasn't changed as much, as we've hit a point in the difficulty charts where the higher-seeded teams that ended up in the losers' brackets earlier are starting to liquidate their opponents in order to try and secure a spot in the Finals.
```

[![](/wiki/shared/news/2024-04-17-osutaiko-world-cup-2024-semifinals-recap/Algorithm_54.jpg)](https://osu.ppy.sh/beatmapsets/2163075#taiko/4561785)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/398/Songs/Sparxe_-_Switcheroo.mp3">
    </audio>
</div>

[![](/wiki/shared/news/2024-04-17-osutaiko-world-cup-2024-semifinals-recap/Switcheroo.jpg)](https://osu.ppy.sh/beatmapsets/2163116#taiko/4561848)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/415/Songs/lemm%20-%20Algorithm%2054.mp3">
    </audio>
</div>

[![](/wiki/shared/news/2024-04-17-osutaiko-world-cup-2024-semifinals-recap/Pomegranate.jpg)](https://osu.ppy.sh/beatmapsets/2163122#taiko/4561855)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/media/news/2024-04-17-pomegranate.mp3">
    </audio>
</div>

## Semifinals results

> Drum roll please... Yeah yeah, funny joke I know. Here are the results.

Reviewer: two authors, so `I` doesn't make much sense

```suggestion
Drum roll please...

Yeah yeah, very funny indeed. Here are the results.
```

| Team A |  |  | Team B |
| --: | :-: | :-: | :-- |
| **China** ::{ flag=CN }:: | **6** | 0 | ::{ flag=FI }:: Finland |
| **Japan** ::{ flag=JP }:: | **6** | 0 | ::{ flag=BR }:: Brazil |
| **Taiwan** ::{ flag=TW }:: | **6** | 0 | ::{ flag=CA }:: Canada |
| **Chile** ::{ flag=CL }:: | **6** | 2 | ::{ flag=FR }:: France |
| **United States** ::{ flag=US }:: | **6** | 1 | ::{ flag=MY }:: Malaysia |
| **United Kingdom** ::{ flag=GB }:: | **6** | 5 | ::{ flag=KR }:: South Korea |
| China ::{ flag=CN }:: | 3 | **6** | ::{ flag=TW }:: **Taiwan** |
| **United States** ::{ flag=US }:: | **6** | 2 | ::{ flag=CL }:: Chile |

## Match highlights

### ::{ flag=GB }:: United Kingdom vs. ::{ flag=KR }:: South Korea

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2119605829&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> There's South Korea. Former podium-placers. A team with several top 50 players and tournament veterans, well-rounded and fit to cover everything the mappools have to offer. Seeding 3rd with balanced placements across the board. Then there's the United Kingdom, the former podium-placers. A team with top 50 players and tournament veterans, well-rounded and fit to…

Reviewer: ```suggestion
There's South Korea. Former podium-placers. A team with several top 50 players and tournament veterans, well rounded and fit to cover everything the mappools have to offer. Seeding 3rd with balanced placements across the board. Then there's the United Kingdom, former podium-placers. A team with top 50 players and tournament veterans, well rounded and fit to…
```

Hey… Wait a minute… Aren't they basically the same? Looking at the stats, the only thing separating these guys' seedings is a rank sum difference of *two*. And you're telling me they're up against each other now? In the winners' bracket semifinals?! In a battle to secure the aforementioned podium placement?! (Localized entirely within your kitchen?!)

> Well you better strap in, because this was exactly the type of roller coaster match you're imagining.

Reviewer: ```suggestion

Well you better strap in, because this was exactly the type of roller coaster match you're imagining.
```

The United Kingdom won the roll and came swinging right out of the gate with [Masayoshi Iimori - NEWALONE (Extended Mix)](https://osu.ppy.sh/beatmapsets/2163092#taiko/4561817), grabbing a quick point thanks to their spectacular accuracy. South Korea countered with [MYUKKE. - Mach Roger](https://osu.ppy.sh/beatmapsets/2163105#taiko/4561831), thinking they could make use of their Hidden trump card, ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027/taiko), but nuh-uh. The UK grabbed that one too, and continued to do so with two more picks, making it 4–0. A gut-wrenching sight for the Koreans.

> The United Kingdom had proven themselves to be *just a cut above*, but when they went fishing in the Free Mod pool with [Sparxe - Switcheroo](https://osu.ppy.sh/beatmapsets/2163116#taiko/4561848), South Korea found the opening they'd been looking for, grabbing a breakpoint and bouncing back with one after another. Some time passed and all of a sudden we were here again, the first tiebreaker since the Round of 32.

Reviewer: ```suggestion
The British had proven themselves to be *just a cut above*, but when they went fishing in the Free Mod pool with [Sparxe - Switcheroo](https://osu.ppy.sh/beatmapsets/2163116#taiko/4561848), the South Koreans found the opening they'd been looking for, grabbing a breakpoint and bouncing back with one after another. Some time passed and all of a sudden we were here again — the first tiebreaker since the Round of 32.
```

> Since we've jumped a few rounds, adding mods to the tiebreaker map is now nothing short of lunacy. But for two of the top 3 seeds and some guaranteed prize money on the line, that didn't seem to matter. A lone Hidden from South Korea, and a Hidden and Hard Rock from the United Kingdom. [Tektheist - Pomegranate](https://osu.ppy.sh/beatmapsets/2163122#taiko/4561855) did all it could to make sure both mods had a horrible time. But again, that didn't seem to matter. The Korean No Mod players ended up taking the brunt of the misses which pushed the lead further and further towards the UK. And despite being put through the wringer for a full five minutes, the United Kingdom made sure it's coming home, together with an unfathomable Hard Rock full combo from ::{ flag=GB }:: [goheegy](https://osu.ppy.sh/users/8057655/taiko).

Reviewer: ```suggestion
Since we've jumped a few rounds, adding mods to the tiebreaker map is now nothing short of lunacy. But for two of the top 3 seeds and some guaranteed prize money on the line, that didn't seem to matter. A lone Hidden from South Korea, and a Hidden and Hard Rock from the United Kingdom. [Tektheist - Pomegranate](https://osu.ppy.sh/beatmapsets/2163122#taiko/4561855) did all it could to make sure both mods had a horrible time. But again, that didn't seem to matter.

The Korean No Mod players ended up taking the brunt of the misses which pushed the lead further and further towards the UK. Not letting up one bit despite being put through the wringer for a full five minutes on the tiebreaker, the United Kingdom made sure to take it home, together with an unfathomable Hard Rock full combo from ::{ flag=GB }:: [goheegy](https://osu.ppy.sh/users/8057655/taiko).
```

---

> We just spoke with Taikonator, and they wanted us to remind you that you can find full details of the TWC on the [official wiki page](/wiki/Tournaments/TWC/2024). Along with that, that they hope they see you in the Twitch chat this weekend over on the [osulive](https://www.twitch.tv/osulive) Twitch channel!

Reviewer: ```suggestion
We just spoke with Taikonator, and they wanted us to remind you that you can find full details of the TWC on the [wiki page](/wiki/Tournaments/TWC/2024). Along with that, that they hope they see you in the Twitch chat this weekend over on the [osulive Twitch channel](https://www.twitch.tv/osulive)!
```
